src/lib.rs

```diff
@@ -586,22 +593,54 @@ mod tests {
     fn parse_csi_max_params() {
         use crate::MAX_PARAMS;
 
-        static INPUT: &[u8] = b"\x1b[1;1;1;1;1;1;1;1;1;1;1;1;1;1;1;1;1;p";
+        // This will build a list of repeating '1;'s
+        // The length is MAX_PARAMS - 1 because the last semicolon is interpreted
+        // as an implicit zero, making the total number of parameters MAX_PARAMS
+        let params = std::iter::repeat("1;").take(MAX_PARAMS - 1).collect::<std::string::String>();
+        let input = format!("\x1b[{}p", &params[..]).into_bytes();
 
         // Create dispatcher and check state
         let mut dispatcher = CsiDispatcher::default();
         assert!(!dispatcher.dispatched_csi);
 
-        // Run parser using OSC_BYTES
+        // Run parser using INPUT
         let mut parser = Parser::new();
-        for byte in INPUT {
-            parser.advance(&mut dispatcher, *byte);
+        for byte in input {
+            parser.advance(&mut dispatcher, byte);
+        }
+
+        // Check that flag is set and thus csi_dispatch assertions ran.
```

Comment:
```suggestion
        // Check that flag is set and thus csi_dispatch assertions ran
```

For consistency I'd remove the dot at the end here, since all the other comments don't seem to have one either.